Competitive_Programming/Python/Greedy/activity_selection.md

```diff
@@ -0,0 +1,107 @@
+# Activity Selection Problem Using Greedy Approach
+
+An <b>activity selection</b> is the problem of scheduling a resource among several competing actvities.
+
+Here, we will use the <b>greedy approach</b> to address this problem.
+
+```
+Greedy Algorithm - It is an algorithmic strategy that makes the best optimal choice at each small stage which will eventually lead to an optimum solution for the entire problem. It always picks the best immediate output unlike dynamic approach which takes all the test cases into consideration.
+```
+
+## Working:
+
+### Solution Approach:
+
+```
+1. Firstly, do the sorting of all activities based on their finishing time.
+2. Select the first activity from the sorted list of activities and print it.
+3. Repeat the same process for the remaining activities.
+```
+
+### Example:
+
+Consider the following list of activities, where the second column represents different activities and the third column represents (start_time, finish_time).
+
+<img src='https://user-images.githubusercontent.com/65852362/141511389-d8de0ade-398f-45eb-9356-9698b68f3b04.png'>
+
+While choosing activities, we need to keep in mind that two activities i and j are compatible if the half-open interval <b>[s<sub>i</sub> >= f<sub>j</sub>)</b> or <b>[s<sub>j</sub> >= f<sub>i</sub>)</b>.
```

Comment:
Can you add a note about what is **_s<sub>i</sub>, f<sub>j</sub>_**...etc
It will be easy to read.